media/js/common/formsets.js

```diff
@@ -31,7 +31,7 @@
  *   'totalForms': '-TOTAL_FORMS',  // End of `name` attr of hidden field indicating total number of
  *                                  // forms (useful if you use multiple formsets in a single <form>)
  * })
- *  
+ *
  */
 
 (function($) {
```

Comment:
We could add a 'use strict' after this line. We'll need to watch those in-case there's other things that get broken by use of strict.